packages/mobile/views/onboarding/views/profile-setup/views/EnterNameView.svelte

```diff
@@ -8,15 +8,15 @@
     import { profileRecoveryRouter, profileSetupRouter } from '@/routers'
     import { onboardingProfile, updateOnboardingProfile } from '@contexts/onboarding'
 
-    const title = localize('views.onboarding.profileSetup.enterName.title', {
-        values: { protocol: formatProtocolName($onboardingProfile?.networkProtocol) },
-    })
-
     let error = ''
     let profileName = $onboardingProfile?.name ?? ''
 
     $: isProfileNameValid = profileName && profileName.trim()
     $: profileName, (error = '') // Error clears when profileName changes
+    $: networkId = $onboardingProfile?.network?.id
+    $: title = localize('views.onboarding.profileSetup.enterName.title', {
+        values: { protocol: formatProtocolName(networkId) },
+    })
```

Comment:
The `networkId` won't change in this view, so these shouldn't be reactive statements.